### PR TITLE
Add Ukrainian localization

### PR DESCRIPTION
**Pull request by dimacheverda**

### CODE REVIEW

**Comment by ArtSabintsev** [comment]
Thanks! Can you also update the Tests?

**Comment by dimacheverda** [comment]
@ArtSabintsev updated the Tests target but buddybuild CI failing in Xcode on '@available' operator, which was introduced later.

**Comment by ArtSabintsev** [comment]
Alright, updated to Xcode 9 in BB for this branch.

File: HarpyExample/HarpyExampleTests/HarpyTests.m [hunk 1]
```diff
@@ -486,6 +486,15 @@ - (void)testTurkishLocalization {
     XCTAssertTrue([[_harpy testLocalizedStringForKey:@"Update"] isEqualToString:@"Güncelle"]);
 }
 
+- (void)testUkrainianLocalization    {
+    [_harpy setForceLanguageLocalization:HarpyLanguage];
```
Comment (ArtSabintsev) [inline HarpyExample/HarpyExampleTests/HarpyTests.m]: This should be `HarpyLanguageUkranian`.

Comment (dimacheverda) [reply HarpyExample/HarpyExampleTests/HarpyTests.m]: Fixed mistyped constant.

**Comment by ArtSabintsev** [comment]
This is live - https://github.com/ArtSabintsev/Harpy/releases/tag/4.1.8

Thanks!

**Pull request merged**